.github/workflows/test.yaml

```diff
@@ -36,5 +36,5 @@ jobs:
           amend: true
         if: |
           matrix.os == 'ubuntu-latest' &&
-          github.event_name == 'push'
+          github.ref == 'refs/heads/main'
```

Comment:
you don't wanna test on PRs ?

Reply:
Not really as the action will update a wiki page in our repo:

https://github.com/marketplace/actions/go-coverage-report

See here for example

https://github.com/ncruces/go-sqlite3/wiki/Test-coverage-report